src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetCommandHandler.cs

```diff
@@ -64,6 +70,65 @@ public async Task<List<PSCommandMessage>> Handle(GetCommandParams request, Cance
             {
                 foreach (dynamic command in result)
                 {
+                    if (!_skipPackageManagementCheck && command.ModuleName == "PackageManagement" && command.Version < new Version(1, 4, 6))
+                    {
+                        _logger.LogDebug("Old version of PackageManagement detected. Attempting to update.");
+
+                        var takeActionText = "Yes";
+                        MessageActionItem messageAction = await _languageServer.Window.ShowMessage(new ShowMessageRequestParams()
+                        {
+                            Message = "You have a version of PackageManagement that causes issues with the PowerShell extension. Would you like to update PackageManagement (You will need to restart the PowerShell extension after)?",
+                            Type = MessageType.Warning,
+                            Actions = new MessageActionItem[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5230911)